server/RenderContext.cs

```diff
@@ -466,8 +466,13 @@ public void Render(XGraphics graphics)
                     if (styles.microBorders.visible)
                     {
                         if (styles.fillMicroBorders)
+                        {
                             DrawMicroBorders(BorderLayer.Fill);
+                            DrawRegions(BorderLayer.Fill);
```

Comment:
In MSEC, borders are stroked (only), regions are filled (only). I guess it depends on what your intent is. Borders as-is don't need to correspond to an allegiance, but there's not a "filled by default option" and also not a "no stroke" option.

Since the TravellerMap default is stroked borders, I'd be fine with them being like MSEC by default (filled, not stroked). But if they're just like borders I don't get the point.